.circleci/config.yml

```diff
@@ -140,7 +140,8 @@ jobs:
               -kubeconfig="$primary_kubeconfig" \
               -secondary-kubeconfig="$secondary_kubeconfig" \
               -debug-directory="$TEST_RESULTS/debug" \
-              -consul-k8s-image=hashicorpdev/consul-k8s:crd-controller-base-latest
+              -consul-k8s-image=hashicorpdev/consul-k8s:crd-controller-base-latest \
+              -consul-image=ashwinvenkatesh/consul:main-oct6@sha256:f34ca59e93c564082a18e18850d6dd8c6ac43b97afbe9bd9183ac4aab15c70b1
```

Comment:
We need to provide our own image here as we rely on features that are currently on the master branch of `consul`